tex/generic/pgf/libraries/shapes/pgflibraryshapes.multipart.code.tex

```diff
@@ -300,6 +300,119 @@
 }%
 
 
+%
+% A diamond that is split in the middle into an upper and a lower part.
+%
+% This node consists of two parts: the upper (text) part is shown in the
+% upper half of the diamond. The second part is the (optional) lower part.
+%
+% Parts: text, lower
+
+\pgfdeclareshape{diamond split}
+{%
+  %
+  % Node parts
+  %
+  \nodeparts{text,lower}
+  %
+  % Anchors
+  %
+  \savedanchor\outernortheast{%
+    %
+    % Calculate width and height of the text rectangle
+    %
+    \pgf@xa=\wd\pgfnodeparttextbox%
```

Comment:
Please remove `%` after all assignments (including `\advance` statements). I know that Till did it that way everywhere, but it's objectively wrong.

Reply:
I agree; that particular line is probably there because I copied that particular line verbatim from either `diamond` or `circle split` and didn't remove that one.